Update functions-bindings-sendgrid.md

simonaco opened the pull request:
Fix from property to match the SendGrid API for nodejs function according to https://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/index.html

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Update functions-bindings-sendgrid.md

Fix from property to match the SendGrid API for nodejs function according to https://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/index.html
